tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -197,6 +198,14 @@ describe(__filename, () => {
     expect(root.find('.AddonReviewListItem-edit')).toHaveLength(0);
   });
 
+  it('assigns isOwner property if you wrote the review', () => {
+    const review = signInAndDispatchSavedReview();
+    const root = render({ review });
+    const rating = root.find(Rating);
+
+    expect(rating).toHaveProp('isOwner', true);
```

Comment:
I would check for the actual output `className` as well, because we want that rendered when this is true.